app/js/app.babel.jsx

```diff
@@ -23,6 +24,7 @@ class API {
     this._tryToRestore();
     this._listenUnload();
 
+    this._drawStartPath();
```

Comment:
The current workflow is:

- The app tries to restore the state from `localStorage` - the `_tryToRestore` method in `app.babel.js`
  - If there is a record in `localStorage` - app parses it and send to the `store`
  - if no record - the store is populated with default path of `2` points - the `reset` function in `app.babel.js`

We need to modify this algorithm to have a `startPath`:

- The app tries to restore the state from `localStorage` - the `_tryToRestore` method in `app.babel.js`
  - If there is a record in `localStorage`:
    - app parses it and send to the `store`
  - else, if no record:
     - if `startPath` is set - parse it and populate the store with the path
     - if `startPath` is not set - call the `reset` function as before

This will ensure that the `startPath` is applied only once - when there is no path state yet. 

Please let me know if you have any qq.